viper/parser.py

```diff
@@ -627,6 +627,8 @@ def parse_expr(expr, context):
             new_unit = combine_units(left.typ.unit, right.typ.unit)
```

Comment:
Actually about L623 above. Does it make sense not to implement cross-type operations? I believe it might for `num256`|`decimal`, as we might have data loss there, but `num`|`num256` can occur fine withouth data loss (implicit conversion of `num` to `num256`).

Same below L656 (in updated file) for division. Does it make sense not to have cross-type division? I'm leaning towards yes, but discuss.

Reply:
@fubuloubu sure I was going to do cross-type operations as a seperate PR. For the time being the compiler will throw an error with any type of cross-type operation. If you prefer I can do `num | num256` as part of this PR/branch ?

Reply:
I think doing it as a separate PR would work best. There are some subtle considerations that I think warrant a second PR. I'll add an issue to remind ourselves of this.

Reply:
Cool thanks :) :+1: